maintenance/health/servicehealthcheck/healthchecker.go

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2019 by PACE Telematics GmbH. All rights reserved.
+// Created at 2019/10/18 Charlotte Pröller
+
+package servicehealthcheck
+
+import (
+	"net/http"
+	"strings"
+	"sync"
+
+	"github.com/pace/bricks/maintenance/errors"
+	"github.com/pace/bricks/maintenance/log"
+)
+
+// HealthCheck is a health check that is initialised once and that is performed
+// periodically and/or spontaneously.
+type HealthCheck interface {
+	HealthCheck() (bool, error)
+}
+
+type Initialisable interface {
+	Init() error
+}
+
+type handler struct{}
+
+// checks contains all registered Health Checks - key:Name
+var checks sync.Map
```

Comment:
Why not move these variables into the `handler` struct?

Reply:
I don't see any improvement in doing this, because then we would have to store the handler as package variable (we need access to initErrors during health check registration). Resolving this as discussed with @Smuerdt